### ISSUE TITLE
Scrolling the hydra hint window

### ISSUE THREAD

**Issue by Ekleog** [abo-abo/hydra#370]
Hello,

I've started using treemacs recently, and when typing `?` its hydra is too wide for my screen.

I couldn't find a way to scroll the hydra hint window, and it looks like trying to use its scrollbar just closes it.

Is there already a way to do this, or is it currently not possible with regular hydra?

Thank you anyway for what looks like a neat library :)

**Comment by SreenivasVRao** [comment]
Yup - seeing the same issue. Source code pro. emacs 27, spacemacs develop,  Font size 12. 24 inch monitor.

**Comment by abo-abo** [comment]
Is there an easy way for me to reproduce the issue? Could you include the steps and the minimal code?

**Comment by Ekleog** [comment]
I have no idea how to write elisp code, sorry :/ but installing treemacs and setting the emacs font to be large enough for the hint window not to fit in a single screen would probably do it -- I would be willing to guess it'd work with any hydra that has enough columns, but can't really test.

**Comment by izzues** [comment]
+1

To reproduce, it's enough to open Hydra and resize your window until it's narrower than the hints frame.

**Comment by Alexander-Miller** [comment]
PSA: The treemacs hydra is split in 2 now.

**Comment by umucorn** [comment]
Same here, I can't scroll it vertically.
